### ISSUE TITLE
Dark Mode > Tab name Channels/Chats becomes invisible in the dark mode

### ISSUE THREAD

**Issue by tlle14** [linagora/Twake-Mobile#895]
# Precondition
Dark mode is on

# Actual Result
The tab name Channel/ Chats becomes invisible in the Dark mode

 # Expected Result
It should visible even in the dark mode
Because the app is not applied the dark mode so the tab should display as normal

# Environment

- Build version: 3.0.5
- Device:
Samsung Galaxy A51 - Android 11 - 6.5 inch
iPhone 11 Promax - iOS 14.4 - 6.46 inch
- QA environment

**Comment by AyoungDukie** [comment]
This appears to occur with the light mode on Android as well:
![Screenshot_20211109-150923_Twake](https://user-images.githubusercontent.com/5449443/140998376-95fe8414-ba75-432d-841a-dcebb5032e58.png)

**Comment by tlle14** [comment]
Thank @AyoungDukie for the report. 
However, I cannot reproduce the issue on my test device (Samsung A51, Android 11) in light mode. 
Could you please provide your device name and model?

**Comment by AyoungDukie** [comment]
I'm on a Pixel 4a also with stock Android 11. 

For grins and giggles re-tried this, and realized I originally had Battery Saver mode on (which I forgot actually turns on Dark mode for Pixels). So I checked - after disabling battery saver and changing back to light mode, - but I did not reload the app and still had the above issue. I did not realize that the theme in the app did not reload, so I needed to close/relaunch the app after changing theme. 

To recap, once I closed the app and relaunched with Light mode already enabled, the text appears. 

However, if battery saver turns back on, it will temporarily enable Dark mode and will produce this issue the next time the app is relaunched.

**Comment by tlle14** [comment]
Retest pass on the newest version 1.0.4 
Hi @AyoungDukie the bug should be fixed in the next release. Please wait for it. 

https://user-images.githubusercontent.com/69950655/149097437-1328a074-f49c-4afd-a4f9-2323fcd1a070.mp4
